workspace.bzl

```diff
@@ -41,4 +41,4 @@ def rust_workspace():
 
     bazel_skylib_workspace()
 
-    bazel_version(name = "bazel_version")
+    bazel_version(name = "io_bazel_rules_rust_bazel_version")
```

Comment:
can you add a TODO referencing #462 to investigate if this is still necessary, and the original issue where you were seeing naming conflicts (#268)?

Reply:
Done.

Reply:
thanks!